examples/long_tutorials/handle_drift/plot_handle_drift.py

```diff
@@ -0,0 +1,342 @@
+"""
+===========================================
+Handle probe drift with spikeinterface NEW
+===========================================
+
+Probe movement is an inevitability when running
+*in vivo* electrophysiology recordings. Motion, caused by physical
+movement of the probe or the sliding of brain tissue
+deforming across the probe, can complicate the sorting
+and analysis of units.
+
+Spikeinterface offers a flexible framework to handle motion correction
+as a preprocessing step. In this tutorial we will cover the three main
+drift-correction algorithms implemented in SpikeInterface
+(**rigid_fast**, **kilosort_like** and **nonrigid_accurate**) with
+a focus on running the methods and interpreting the output.
+
+For more information on the theory and implementation of these methods,
+see the :ref:`motion_correction` section of the documentation.
+
+---------------------
+What is probe drift?
+---------------------
+
+The inserted probe can move from side-to-side (*'x' direction*),
+up-or-down (*'y' direction*) or forwards-or-backwards (*'z' direction*).
+Movement in the 'x' and 'y' direction is harder to model than vertical
+drift (i.e. along the probe depth), and are not handled by most motion
+correction algorithms. Fortunately, vertical drift which is most easily
+handled is most pronounced as the probe is most likely to move along the path of insertion.
+
+Vertical drift can come in two forms, *'rigid'* and *'non-rigid'*. Rigid drift
+is drift caused by movement of the entire probe, and the motion is
+similar across all channels along the probe depth. In contrast,
+non-rigid drift is instead caused by local movements brain along the
+probe, and can selectively affect subsets of channels..
+
+--------------------------
+The drift correction steps
+--------------------------
+
+The easiest way to run drift correction in SpikeInterface is with the
+high-level :py:func:`~spikeinterface.preprocessing.correct_motion()` function.
+This function takes a preprocessed recording as input and returns a motion-corrected
```

Comment:
Good point, canonically it should take a recording that has been at least filtered. But the type of preprocessing is unspecific, and it will work on any recording, so I removed 'preprocessed' here. There is a more detailed discussion on the preprocessing later anyway!